week-4/Homework/mandatory/4-groceries.js

```diff
@@ -39,6 +41,19 @@ Exercise 2:
 */
 // Gather weekend item names into this array
 let weekendGroceriesToBuy = [];
+ for(let weekendDay in weeklyMealPlan){
+  if(weekendDay === 'saturday' || weekendDay === 'sunday'){
+  
+    for(let i = 0; i < weeklyMealPlan[weekendDay].length; i++){
+      weekendGroceriesToBuy.push(weeklyMealPlan[weekendDay][i]);
+      
+    }
+    
+  }
+
+}
+console.log(`${weekendGroceriesToBuy}`);
```

Comment:
Beware using `console.log` with a string template like this as it will not print the array out in the way you expect 
You can do `console.log(weekendGroceriesToBuy)` and this will work fine 
```suggestion
console.log(weekendGroceriesToBuy);
```